src/types/rest/response/fills.ts

```diff
@@ -0,0 +1,29 @@
+import { OrderSide } from "./orders";
+
+export interface Fill {
+  fee: number;
+  feeCurrency: string;
+  feeRate: number;
+  future: string;
+  id: string;
+  liquidity: "maker" | "taker";
+  market: string;
+  baseCurrency?: string | null;
```

Comment:
I've seen a few like this, just to check - are they really undefined or string or null? all 3 possibilities?